### ISSUE TITLE
Problem with "In war" status of The Scryers faction

### ISSUE THREAD

**Issue by KazidosWoW** [gamefreedomgit/Maelstrom#5904]
**Description:**

Hi. 
I got exalted with Aldor first and after with The Scryers, to get the achievement and Feats of Strength about. 

![Captura de pantalla 2024-04-21 190654](https://github.com/gamefreedomgit/Maelstrom/assets/166726682/39e6694c-a559-4af7-99b8-e04a0e984a84)
![Captura de pantalla 2024-04-21 190717](https://github.com/gamefreedomgit/Maelstrom/assets/166726682/719a58a7-8df4-488c-a31b-316ffe79bff0)

https://www.wowhead.com/tbc/faction=934/the-scryers
https://www.wowhead.com/tbc/faction=932/the-aldor

https://www.wowhead.com/achievement=903/shattrath-divided
https://www.wowhead.com/achievement=1205/hero-of-shattrath

I decided to keep exalted with The Scryers and hated with Aldor, the problem is that I appear " in war" status with both factions. I can´t interact with Scryers due to this bug. "In war" status can´t be activated by player in this two factions and i can´t switch off the option. It shoudnt happen, is neccesary to interact normally with the faction you prefer, in this case The Scryers, for get others quests, reputation, mounts, tier trasmog, etc,..

![Captura de pantalla 2024-04-21 190907](https://github.com/gamefreedomgit/Maelstrom/assets/166726682/5d4f74a6-72bd-43d0-b997-e6d3c4184092)

**How it should work:**

When u get exalted with one of this faction, the enemy faction will be hated with you, and you only can interact with the faction you choose to get exalted. But this option "in war" can´t be activated or desactivated by players, and the game shouldnt mark them in war being exalted.

Thank you so much.
Regards, Kazidos.

**Comment by UltraNix** [comment]
@KazidosWoW How you were able to earn exalted with Scryers if you were already at war with them?

**Comment by KazidosWoW** [comment]
I get exalted with Scryers before being in war, using repeatedly the next quest:  https://www.wowhead.com/tbc/quest=10025/more-basilisk-eyes  
and continue with NetherStorm zone quests.

I think that this problem perhaps started when i get being exalted and have this quest in progress:

https://www.wowhead.com/quest=10507/turning-point

I didnt have problem to take the quest but when I completed the objetive and flied to Shattrath to delivery the quest to get the gold reward I saw I can't interact with the Scryers faction, who gave me the quest, and reviewing reputations I see this bug.

Now, how u can appreciate in the image, I have Aldor  (hated) and Scryers (exalted) in War status, and is impossible to activate or desactivate manually by any player.
